common/scala/src/main/scala/org/apache/openwhisk/core/entity/LogLimit.scala

```diff
@@ -48,12 +48,15 @@ protected[core] class LogLimit private (val megabytes: Int) extends AnyVal {
 protected[core] object LogLimit extends ArgNormalizer[LogLimit] {
   val config = loadConfigOrThrow[MemoryLimitConfig](ConfigKeys.logLimit)
 
-  protected[core] val minLogSize: ByteSize = config.min
-  protected[core] val maxLogSize: ByteSize = config.max
-  protected[core] val stdLogSize: ByteSize = config.std
+  protected[core] val MIN_LOGSIZE: ByteSize = config.min
+  protected[core] val MAX_LOGSIZE: ByteSize = config.max
+  protected[core] val STD_LOGSIZE: ByteSize = config.std
+
+  /** A singleton LogLimit with default value */
+  protected[core] val defaultLogLimit = LogLimit(STD_LOGSIZE)
```

Comment:
You named the new singletons `default...Limit` - while this value we used `STD` (standard) in the past. And the term `standard` / `STD` is still used - also in the application config.

For this reason, I think that the new singletons should be named `standard...Limit`. We can keep the term `default` in the comment to clarify.